src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.csproj

```diff
@@ -7,6 +7,7 @@
       disable NU1701 that we are getting for Microsoft.Net.Compilers.Targets.NetCore
       as we know it is safe to resolve its assets as net461.
     -->
+    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">$(PackageTargetFallback);dnxcore50;portable-net45+win8</PackageTargetFallback>
```

Comment:
NIT: can you instead move this new line after the `<NoWarn>.....` so that the comment above stays with the line that it is talking about? Also, it would probably be nice to just add a one line comment above this new line saying that what is causing the problem is the .NETCore.Targets package.

Reply:
@joperezr do you know why this is needed at all? I would only expect this to be an issue on older SDK/CLI versions.

Reply:
Exactly, and I believe CoreCLR's release/2.0 branch is using an older version of the CLI which is why this is an issue here, but it doesn't repro in master (master has 2.0 RTM CLI already)